metrics/error-report/index.js

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/prefer-default-export */
+/* eslint-disable no-prototype-builtins */
+import AES from "crypto-js/aes";
+import Utf8 from "crypto-js/enc-utf8";
+import ScriptLoader from "../../integrations/ScriptLoader";
+// import RudderApp from "../../utils/RudderApp";
+
+const metaData = {
+  SDK: {
+    name: "RudderStack JavaScript SDK",
+    version: "process.package_version",
+    installType: process.browser ? "CDN" : "NPM",
+  },
+};
+
+const credentials = {
+  apiKey:
+    "U2FsdGVkX182Myeej4DKayK1qKvTaxfa/9g4oz/kMD1hkuPCBKmyrWZB74x3N0qf6xuPtH8QRq4YRnIh15AaNg==", // "0d96a60df267f4a13f808bbaa54e535c"
+  appVersion: "1.0.0",
+  releaseStage: "development",
+  key: "Rudder",
+};
+
+const sdkNames = [
+  "browser.js",
+  "rudder-analytics.min.js",
+  "rudder-analytics-staging.min.js",
+  "rudder-analytics.js",
```

Comment:
The errors can also originate from the integration SDKs.

Reply:
For v1 everything is part of SDK. Will add v1.1 specific code in the production-staging branch.

Reply:
We can also drive it from here, but I'm curious about the plan for this.

Reply:
Will import `configToIntNames` and create an array with the values.